public/javascripts/addOrDeleteButton.js

```diff
@@ -58,17 +59,20 @@ var addOrDeleteButtonComponent = (function() {
                             window.location.href = '/user/login'
                         } else {
                             ctrl.addOrDelete()
-                            // $(this).tooltip('fixTitle');
-                            // $(this).tooltip('show');
+                            $(this).tooltip('fixTitle');
```

Comment:
Add a brief comment and maybe a link so that we can refer to why it needs to be this way in the future.